share/pot/set-status.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+# shellcheck disable=SC3033,SC3040,SC3043
+:
+
+: "${_POT_INTERNAL_STATUS:="starting started stopping stopped"}"
+
+set-status-help()
+{
+	cat <<-"EOH"
+	Internal command, DO NOT USE IF YOU DON'T KNOW WHAT YOU ARE DOING!
+	This command is meant to be invoked using lockf
+	pot set-status [-hv] [-p pname] [-s status]
+	  -h print this help
+	  -v verbose
+	  -p pname : pot name
+	  -s status : the status [starting started stopping stopped]
+	EOH
+}
+
+pot-set-status()
+{
+	local _pname _new_status _current_status _conf
+	_pname=""
+	_new_status=""
+	OPTIND=1
+	while getopts "hvp:s:" _o ; do
+		case "$_o" in
+		h)
+			set-status-help
+			return 0
+			;;
+		v)
+			_POT_VERBOSITY=$(( _POT_VERBOSITY + 1))
+			;;
+		p)
+			_pname="$OPTARG"
+			;;
+		s)
+			# shellcheck disable=SC2086
+			if _is_in_list "$OPTARG" $_POT_INTERNAL_STATUS ; then
+				_new_status="$OPTARG"
+			else
+				_error "$OPTARG is not a valid status"
+			fi
+			;;
+		*)
+			set-status-help
+			return 1
+			;;
+		esac
+	done
+	if [ -z "$_pname" ]; then
+		_error "A pot name is mandatory"
+		set-status-help
+		return 1
+	fi
+	if ! _is_pot "$_pname"; then
+		_error "$_pname is not a pot"
+		return 1
+	fi
+
+	_current_status=$(_get_conf_var "$_pname" "pot.status")
```

Comment:
Maybe having something like pot.data would make sense?

Reply:
good point. If the machine crashes, the status will be inconsistent.
I guess it could be a follow up activity, there is the need to manage the new files or folder and so on, but it should be implemented soon.
A jail supervisor is somehow needed, but it's something for later (or another life :) )

Reply:
@pizzamig We should at least think about the various failure scenarios to make people sure/their pots won't get stuck.

Maybe it would be better to store this in a temporary file, like other things like ".pot_main_pid" (from the signal patch) and then provide a simple command that can be used to remove all of pot's temporary files.

Reply:
Using a different file to store the status
I'm unsure about mechanisms to fix situations.
In case of a crash, it's enough to remove the files.
In other cases, you can use the `pot set-status` to fix it, but I would consider it dangerous...